releases/release-2.1.16.md

```
@@ -14,53 +14,53 @@ TiDB Ansible バージョン: 2.1.16
-    -   AUTO_INCREMENT列がFLOATまたはDOUBLE型処理できない問題を修正 [＃11389](https://github.com/pingcap/tidb/pull/11389)
-    -   `CONVERT_TZ`関数が無効な引数を受け入れたときに`NULL`が正しく返されない問題を修正しました [＃11357](https://github.com/pingcap/tidb/pull/11357)
-    -   `PARTITION BY LIST`文でエラーが報告される問題を修正しました。(現在は構文のみがサポートされています。TiDBが文を実行すると、通常のテーブルが作成され、プロンプトメッセージが表示されます) [＃11236](https://github.com/pingcap/tidb/pull/11236)
-    -   `Mod(%)` `Multiple(*)`演算で、 `Minus(-)`以下の桁数が多い場合（ `select 0.000 % 0.11234500000000000000`など）に MySQL の結果と矛盾する`0`結果が返される問題を修正しました[＃11353](https://github.com/pingcap/tidb/pull/11353)
+    -   `INTERVAL`が負の場合に`DATE_ADD`関数が間違った結果を返す問題を修正しました [#11616](https://github.com/pingcap/tidb/pull/11616)
+    -   `DATE_ADD`関数が`FLOAT` 、 `DOUBLE` 、または`DECIMAL`型の引数を受け入れるときに型変換を誤って実行するため、誤った結果が返される可能性がある問題を修正しました[#11628](https://github.com/pingcap/tidb/pull/11628)
+    -   CAST(JSON AS SIGNED)がオーバーフローしたときにエラーメッセージが不正確になる問題を修正しました [#11562](https://github.com/pingcap/tidb/pull/11562)
+    -   1 つの子ノードが閉じられず、Executor を閉じる処理中にエラーが返された場合に、他の子ノードが閉じられない問題を修正しました。 [#11598](https://github.com/pingcap/tidb/pull/11598)
+    -   タイムアウトまでにリージョン分散のスケジュールが完了していない場合に、エラーではなく、正常に分割されたリージョンの数と完了したパーセンテージを返す`SPLIT TABLE`ステートメントをサポートします。 [#11487](https://github.com/pingcap/tidb/pull/11487)
+    -   MySQL との互換性を保つために、 `REGEXP BINARY`関数で大文字と小文字を区別する [#11505](https://github.com/pingcap/tidb/pull/11505)
+    -   `DATE_ADD` / `DATE_SUB`の結果の`YEAR`の値が 0 より小さいかより大きい場合にオーバーフローするため、 `NULL`が正しく返されない問題を修正しました。 [#11477](https://github.com/pingcap/tidb/pull/11477)
+    -   スロークエリテーブルに、実行が成功したかどうかを示す`Succ`フィールドを追加します[#11412](https://github.com/pingcap/tidb/pull/11421)
```

markdownlint-cli2 (0.23.2)

[warning] Spaces after list markers
Expected: 1; Actual: 3

(MD030, list-marker-space)